test/api/statistics.js

```diff
@@ -0,0 +1,104 @@
+'use strict'; /*jslint mocha:true, expr:true */
+
+
+var node = require('./../node.js');
+
+describe('Statistics API', function() {
+
+    /*Define functions for use within tests*/
+    function getLastBlock(done) {
+        node.get('/api/statistics/getLastBlock', done);
+    }
+
+    function getBlocks(done) {
+        node.get('/api/statistics/getBlocks', done);
+    }
+
+    function getPeers(done) {
+        node.get('/api/statistics/getPeers', done);
+    }
+
+    function checkPeersList(id) {
+        for (var i = 0; i < id.length; i++) {
+            if (id[i + 1]) {
+                node.expect(id[i]).to.have.any.keys(
```

Comment:
test fails with all keys because knowlege only works on delegates and known accounts.